ManageWikiExtensions.php

```diff
@@ -2571,6 +2571,32 @@
 				]
 			]
 		],
+	        'wikibasequalityconstraints' => [
+			'name' => 'WikibaseQualityConstraints',
+			'linkPage' => 'https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:WikibaseQualityConstraints',
+			'var' => 'wmgUseWikibaseQualityConstraints',
+			'conflicts' => false,
+			'requires' => [],
+			'install' => [
+				'sql' => [
+					'wbqc_create' => "$IP/extensions/WikibaseQualityConstraints/repo/sq/create_wbqc_constraints.sql",
+					'wbqc_constraint_id' => "$IP/extensions/WikibaseQualityConstraints/repo/sq/patch-wbqc_constraints-constraint_id.sql",
+					'wbqc_constraints_guid_uniq' => "$IP/extensions/WikibaseQualityConstraints/repo/sq/patch-wbqc_constraints-wbqc_constraints_guid_uniq.sql",
```

Comment:
using the wrong sql path. Should be `$IP/extensions/WikibaseQualityConstraints/sql/*` not `$IP/extensions/WikibaseQualityConstraints/repo/sq/*`

Reply:
Oops...